index.js

```diff
@@ -5,46 +5,69 @@ const ora = require('ora');
 const chalk = require('chalk');
 const prettyMs = require('pretty-ms');
 
-const getMetaTag = (html, property) => {
+const getMetaTag = (html, property, spinner) => {
```

Comment:
We shouldn't be passing property/spinner through here and manipulating the spinner inside the funtion. Instead the function should throw/return false and then handle that result elsewhere in the code.